src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -365,7 +351,7 @@ fun finnTssIdFraInfotrygdRespons(sisteSmPeriode: TypeSMinfo.Periode?, behandler:
 
 fun validerMottattSykmelding(helseOpplysningerArbeidsuforhet: HelseOpplysningerArbeidsuforhet): ValidationResult {
     return if (helseOpplysningerArbeidsuforhet.medisinskVurdering.hovedDiagnose == null) {
-        RULE_HIT_STATUS_COUNTER.labels("HOVEDDIAGNOSE_MANGLER").inc()
+        RULE_HIT_STATUS_COUNTER.labels("MANUAL_PROCESSING").inc()
```

Comment:
Tusen takk for at du fikset den her!! 😄 Så det i går da jeg skulle dra at det var noe rart med grafanaboardet for infotrygdregler 😛

Reply:
Null stress, fort gjort.